modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc

```diff
@@ -11,9 +11,14 @@ The console is only available after the {app-security-service} Service has been
 
 .Overview of the Mobile Security Console
 
-The {app-security-service} Console allows you to monitor applications, their respective versions, and disable versions of those applications.
+The {app-security-service} Console allows you to monitor mobile apps, deployed versions, and disable versions of those mobile apps.
 
-Opening the console lists all applications. Click on an application to view version details for that application.
+Opening the console lists all applications. Click on an application to view deployed versions of that mobile app.
+
+A deployed version is registered with Mobile Security after:
+
+* A version of the app is released, for example, 1.2.3.
```

Comment:
Using an unordered list suggests only **one** of these two events need to occur in order for a deployed version to be registered.

The actual case being that point 1 is a requirement for point 2 to be possible.

Do you agree? Maybe this should be an ordered list or some other alternative?

Reply:
I don't think an unordered  list implies OR, but lemme try to make it more explicit ;)